### PR TITLE
APPS-1004 Branch for debugging / testing sbt upgrade.

### PR DESCRIPTION
**Pull request by r-i-v-a**
Branch for debugging / testing sbt upgrade.

Revert "APPS-1004 Update Scala projects to sbt v1.5.7. (#265)"
This reverts commit d90a8f8cfba66d282459a757441b725f4094ea8d. @jdidion  @commandlinegirl

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -33,7 +33,7 @@ lazy val global = root
       settings,
       publish / skip := true
   )
-  .disablePlugins(AssemblyPlugin)
+//  .disablePlugins(AssemblyPlugin)
```
Comment (r-i-v-a) [inline build.sbt]: This line wasn't changed recently, but why was it set this way? (Getting rid of it doesn't seem to help with the current issue.) @jdidion

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.0")
```
Comment (r-i-v-a) [inline project/plugins.sbt:1]: From https://github.com/sbt/sbt-assembly/releases

File: build.sbt (hunk 1 above)
Comment (jdidion) [reply build.sbt]: You need to skip assembly on the global project or you'll get a basically empty JAR created for the root project.

Comment (r-i-v-a) [reply build.sbt]: Aha, thanks @jdidion

**Review by jdidion — DISMISSED** [review]

**Review by jdidion — APPROVED** [review]

**Pull request merged**